### ISSUE TITLE
False positive for `WPS529`

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#3501]
Code like this

```python
def func(**kwargs):
    if 'a' in kwargs:
        assert 'a_kwargs' not in kwargs
    else:
        kwargs['a'] = other(
            **kwargs.pop('a_kwargs', {}),
        )
```

raises `WPS529`

I don't think that it is correct, because we can't replace it with `.get`.

```python
    # Correct:
    value = collection.get(key)
    if value is not None:
        print(value)

    # Wrong:
    if key in collection:
        print(collection[key])
```

**Comment by vldmrdev** [comment]
@sobolevn is this issue still relevant? I can take on this task.

**Comment by sobolevn** [comment]
I can't find any changelog entries about this task. Please, test the `master` version to double check.
